Tasks/AzureResourceManagerTemplateDeploymentV3/task.json

```diff
@@ -14,8 +14,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 3,
-        "Minor": 175,
-        "Patch": 1
+        "Minor": 176,
```

Comment:
I hope this version is bigger than the test package you might have generated for ad-hoc testing for customer.

Reply:
yes, i provided them 3.175.2.